pyproject.toml

```
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=41", "wheel", "setuptools-git-versioning"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name="micro-manager-precice"
+dynamic = [ "version" ]
+dependencies = [
+    "pyprecice>=3.0.0.0", "numpy", "mpi4py", "scikit-learn"
```

You could request a compatible release instead.

```suggestion
    "pyprecice~=3.0", "numpy", "mpi4py", "scikit-learn"
```

---

I could not find a reference to this syntax. How does it change what was happening before?

---

https://peps.python.org/pep-0440/#compatible-release